cmd/zfs/zfs_main.c

```diff
@@ -391,6 +403,10 @@ get_usage(zfs_help_t idx)
 	case HELP_REDACT:
 		return (gettext("\tredact <snapshot> <bookmark> "
 		    "<redaction_snapshot> ..."));
+	case HELP_JAIL:
+		return (gettext("\tjail <jailid|jailname> <filesystem>\n"));
+	case HELP_UNJAIL:
+		return (gettext("\tunjail <jailid|jailname> <filesystem>\n"));
```

Comment:
We'll need the updated man pages for this feature.  Now that the man pages have been split by subcommand a `man/man8/zfs-jail.8` would make good sense.